docs/src/main/asciidoc/spanner.adoc

```diff
@@ -84,47 +235,66 @@ public class MyApplication {
 
 	public void demo() {
 
-	  // storing new students
-	  SimpleStudent ss = new SimpleStudent();
-	  ss.name = "student1";
-	  ss.age = 99;
-	  spannerOperations.insert(ss);
+		this.tradeRepository.deleteAll(); //defined on CrudRepository
```

Comment:
Could we also remove the `@EnableSpannerRepositories` above?
I understand it's not really related to this PR...